src/arrays.js

```diff
@@ -1,61 +1,65 @@
 const getNthElement = (index, array) => {
-  // your code here
+index %= array.length;
+return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split (',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+ return strings.map(strings => strings.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+return strings.map(string => string.split("").reverse().join(""));
 };
 
 const onlyEven = numbers => {
-  // your code here
+return numbers.filter(number => number % 2 === 0);
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+const array2 = [...array];
+array2.splice(index, 1);
+return array2;
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+regex = /^[aeiou]/gi
+return strings.filter(string => string.match(regex));
```

Comment:
You could put this down into one line:
```
  return strings.filter(string => string.match(/^[aeiou]/gi));
```
Remember when creating a variable you have to use either `let` or `const` too!